vendor-extra/ViewsBundle/tests/Views/ViewConverterRegistryTest.php

```diff
@@ -67,28 +67,38 @@ public function it_delegates_to_visitors() : void
 
         $handler->add(
             new CallbackVisitor(
-                function (Element $object, View $view, array &$context = []) : View {
-                    return $view->withTemplate($view->getTemplate().'foo')->withArgument('foo', 'foo');
+                function (Element $object, View $view) : View {
+                    return $view
+                        ->withTemplate($view->getTemplate().'foo')
+                        ->withArgument('foo', 'foo')
+                        ->withContext(['one' => 'foo']);
```

Comment:
isn't it weird to replace everything inside a view? Should return a new one?

Reply:
Only `withTemplate` overwrites, the other two are merged in.

Reply:
(potential naming inconsistency? It's probably a base type used often enough that usage will become clear anyway)